packages/hop-node/src/gasboost/GasBoostTransaction.ts

```diff
@@ -374,7 +385,8 @@ class GasBoostTransaction extends EventEmitter implements providers.TransactionR
   }
 
   async getBumpedGasFeeData (multiplier: number = this.gasPriceMultiplier): Promise<Partial<GasFeeData>> {
-    const use1559 = await this.is1559Supported() && !this.gasPrice
+    const use1559 = await this.is1559Supported() && !this.gasPrice && this.type !== 0
```

Comment:
Technically if `use1559` should always be false if `tyis.type !== 0`, so maybe it is `const use1559 = (await this.is1559Supported() && !this.gasPrice) || this.type !== 0`

Reply:
Updated the comment from `this.type !== 0` to `this.type ===0`

Reply:
> Technically if use1559 should always be false if this.type === 0

That's what the `&& this.type !== 0` is doing

![DeepinScreenshot_select-area_20220124130712](https://user-images.githubusercontent.com/168240/150864981-3fafba2d-6762-45ab-8740-25cd7e225fc7.png)

so I think it's correct